gef-ui/templates/partials/amendments/facility-value.njk

```diff
@@ -0,0 +1,84 @@
+{% extends "index.njk" %}
+{% extends "index.njk" %}
```

Comment:
Minor. Is the second extends a duplicate and redundant?

Reply:
Thanks nice spot :)